### ISSUE TITLE
Support inferring the value for $(Version) from the project being packaged

### ISSUE THREAD

**Issue by terrajobst** [nuproj/nuproj#20]
In many cases, people just package a single class library. And even in cases where the package multiple class libraries, their versions often match anyways.

It's desirable to make it easier to provide a value for `$(Version)` that doesn't force folks to customize their build process.

**Comment by Michael-DST** [comment]
I would like to add that after checking out #68 I am completely happy with the implementation/direction.  I can definitely can work with that!

**Comment by ravensorb** [comment]
Are there any examples on how to use this?

**Comment by kovalikp** [comment]
Sorry, there isn't. #68 is meant to implement this, but that PR is so far behind that I'm kind of scared to look at it. If @williamb1024 would be willing to rebase it on the master, It would be great. But not sure if he would be willing after us letting it rot for so long.

**Comment by williamb1024** [comment]
I would at some point, but #59 has to be dealt with simultaneously, otherwise the builds generate packages that will never be removed due to improper clean targets. I don't remember exactly what the problem was, but not doing partial cleaning caused some type of problem. As I recall Terajobst had some kind of issue with adding support for incremental cleaning.

**Comment by kovalikp** [comment]
I've been working on including `Microsoft.Common.targets` (for completely different issue), which adds support for incremental cleaning and copy to output dir out of the box. Merging this would supplant #59, but it still does not resolve issue it was originally intended to solve :blush:

**Comment by ravensorb** [comment]
Has there been any movement on supporting this request? Even if this PR isn't going to work, this feature would be a HUGE help in managing the nuspec files.

**Comment by AArnott** [comment]
We *do* need to work through our backlog of PRs. I can't promise when it will happen though.
It's whenever we get a chance.

**Comment by donniefitz2** [comment]
Did this ever happen? I'm really liking this project and it does everything I want, except allow me to grab the version number from my assembly and assign it to the NuGet package. Is that possible?

**Comment by AArnott** [comment]
No. For myself I use Nerdbank.GitVersioning on both the class library and the NuProj so the versions always match exactly anyway. I don't know if that will help you get by without this feature.
